.github/workflows/dependabot-auto-approve-minor.yml

```
@@ -0,0 +1,39 @@
+name: Dependabot auto-approve minor updates
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    strategy:
+      matrix:
+        dependencyStartsWith:
+          - '@sentry/'
+          - pg
+          - debug
+          - ethers
+          - typescript
+          - postgrator
+          - '@types/'
+          - standard
+          - prettier
+          - '@sinclair/typebox'
+          - multiformats
```

```suggestion
          - typescript
          - '@types/'
          - neostandard
          - prettier
          - eslint
```

I am not exactly sure if you were going to add those packages in the future, but I'd leave them out for the time being.

---

Thanks for catching this! I copy & pasted this GHA file from another Spark repo. 🙈